engine/apps/oss_installation/cloud_heartbeat.py

```diff
@@ -102,9 +107,16 @@ def send_cloud_heartbeat():
     logger.info("Finish send cloud heartbeat")
 
 
-def get_heartbeat_link(connector, heartbeat):
+def get_heartbeat_link(
+    organization: "Organization",
+    connector: typing.Optional["CloudConnector"],
+    heartbeat: typing.Optional["CloudHeartbeat"],
+) -> typing.Optional[str]:
     if connector is None:
         return None
     if heartbeat is None:
         return None
-    return urljoin(connector.cloud_url, f"a/grafana-oncall-app/?page=integrations&id={heartbeat.integration_id}")
```

Comment:
this previously had a bug in it, `?page=<page>` URL format is no longer supported